modules/stochastic_tools/test/include/vectorpostprocessors/GaussianProcessTester.h

```diff
@@ -0,0 +1,46 @@
+//* This file is part of the MOOSE framework
+//* https://www.mooseframework.org
+//*
+//* All rights reserved, see COPYRIGHT for full restrictions
+//* https://github.com/idaholab/moose/blob/master/COPYRIGHT
+//*
+//* Licensed under LGPL 2.1, please see LICENSE for details
+//* https://www.gnu.org/licenses/lgpl-2.1.html
+
+#pragma once
+
+// MOOSE includes
+#include "GeneralVectorPostprocessor.h"
+#include "SamplerInterface.h"
+#include "SurrogateModelInterface.h"
+#include "GaussianProcess.h"
+
+/**
+ * A tool for output Gaussian Process Surrogate data.
+ */
+class GaussianProcessTester : public GeneralVectorPostprocessor,
```

Comment:
Let's derive this from SurrogateTester. The validParams and constuctor are identical to it, except for the declaration of the std vector.

Reply:
I have done this but it is not the cleanest